test/functional/feature_nulldummy.py

```diff
@@ -82,31 +82,59 @@ def run_test(self):
         self.lastblocktime = int(time.time()) + self.lastblockheight
 
         self.log.info(f"Test 1: NULLDUMMY compliant base transactions should be accepted to mempool and mined before activation [{COINBASE_MATURITY + 3}]")
-        test1txs = [create_transaction(self.nodes[0], coinbase_txid[0], self.ms_address, amount=49)]
+        input = {"txid": coinbase_txid[0], "vout": 0}
+        output = {self.ms_address: 49}
+        rawtx = self.nodes[0].createrawtransaction([input], output)
+        signedtx = self.nodes[0].signrawtransactionwithkey(hexstring=rawtx, privkeys=[self.nodes[0].PRIV_KEYS[0][1]])
+        test1txs = [tx_from_hex(signedtx["hex"])]
```

Comment:
I am wondering if it helps if there was a local `create_transaction` helper? As this is creating a simple chain of txs, the vout could be hardcoded to `0` in the helper, also the node could be hardcoded to `self.nodes[0]`, as there is only one node.

So you'd call `self.create_transaction(coinbase_txid[0], self.ms_address, amount=49, privkey=self.nodes[0].PRIV_KEYS[0][1])` here.

The benefit would be less code and less symbols in this scope.

Thoughts?